accepted/vulnerability-response-runbook.md

```diff
@@ -0,0 +1,203 @@
+giving a stakeholder complete access to the advisory draft and patch, or providing only
+more limited information, such as a warning that a vulnerability exists and a public
+disclosure is expected on a given date.
+
+Bytecode Alliance members are encouraged to socialize which projects they rely on
+in production so that incident managers know to consider them for advanced disclosure.
+
+No individual or orginization is entitled to advanced disclosure. Advanced disclosure is
+not guaranteed to orginizations by membership in the Bytecode Alliance, nor is it exclusive
+to Bytecode Alliance members. Selecting stakeholders for advanced disclosure appropriately
+is critical for maintaining trust in the Bytecode Alliance and its projects.
+
+## Commit to a public disclosure date
+
+Once you have confidence in a patch, and stakeholders have been pre-notified, it is
+time to determine a date for public disclosure. Incident managers may discuss the public
+disclosure date with stakeholders who have recieved advanced to understand their feasibility
+and timelines for mitigation.
+
+By default, public disclosure should be no more than 30 days after initial discovery
+of the vulnerability. Incident managers are encouraged to select the shortest timeline
+possible.
+
+However, the incident manager may make an exception to the 30 day limit if the
+nature of the vulnerability requires it. Reasons for exceptions may include the
+technical feasibility of mitigating the vulnerability, or coordination with other
+security disclosure processes. Do not make exceptions lightly: inappropriate exceptions
+may undermine trust in the Bytecode Alliance and its projects.
+
+## Notify sec-announce 
+
+Once you have committed to a disclosure date, send an email to the
+[`sec-announce@bytecodealliance.org` Google
+Group](https://groups.google.com/a/bytecodealliance.org/g/sec-announce).
+
+This email should use this template:
+
+```pre
+The Bytecode Alliance would like to announce the forthcoming security release
+of the <project>.
+
+The release will be made available on <date> at approximately <time> UTC.
+Additionally, an advisory will be made available on <date> at approximately
+<time> at https://github.com/advisories.
+
+The highes severity issue fixed in this release is <severity>, based on the
+classification scheme defined in the OpenSSL Security Policy.
+
+<non-affected projects> Bytecode Alliance projects are unaffected.
+```
+
+
+## Finalize the security advisory
+
+Improve the draft description of the security advisory into a public facing
+document.
+
+Security Advisories may not assign blame to individuals or orginizations
+who introduced or contributed to the vulnerability.
+
+Security Advisories may credit individuals who contributed to discovering and
+remediating the vulnerability. Use the credit feature of the GitHub Security
+Advisory to do so. This feature requires individuals to consent to being
+credited through their GitHub account.
```

Comment:
If the individual who discovered the vulnerability doesn't have a github account what is the intended procedure? Provide their name and or website in the security advisory body? Or something else?